CRM/Contribute/Page/ContributionRecur.php

```diff
@@ -107,7 +100,7 @@ public function preProcess() {
   public function run() {
     $this->preProcess();
 
-    if ($this->_action & CRM_Core_Action::VIEW) {
+    if ($this->isViewContext()) {
```

Comment:
@mattwire outside the scope of this PR but I'd like to consider moving this function up to CRM_Core_Form / CRM_Core_Page level as it's much more readable

Reply:
Fine by me :-)